src/controller/DevicePairingDelegate.h

```diff
@@ -70,6 +70,17 @@ class DLL_EXPORT DevicePairingDelegate
      *   Called when the commissioning process is complete (with success or error)
      */
     virtual void OnCommissioningComplete(NodeId deviceId, CHIP_ERROR error) {}
+
+    /**
+     *   Called when the commissioning process is complete (with success or error)
+     *   Should be override by PairingDelegates which are JF compatible
+     *   - trustedIcacPublicKeyB has a value only when Joint Commissioning Method is executed
+     */
+    virtual void OnCommissioningComplete(NodeId deviceId, Optional<Crypto::P256PublicKey> trustedIcacPublicKeyB, CHIP_ERROR error)
```

Comment:
Should this really be passing the entire public key by value, not by const reference?